extensions/wikia/ContributionPrototype/scripts/LoadEditor.js

```diff
@@ -0,0 +1,31 @@
+require(['jquery', 'wikia.window'],
+	function ($, window) {
+		'use strict';
+
+		$(function() {
+			console.log("CP load editor");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS2228)